draft-ietf-quic-recovery.md

```diff
@@ -1067,14 +1078,15 @@ Pseudocode for OnLossDetectionTimeout follows:
 
 ~~~
 OnLossDetectionTimeout():
-  if (crypto packets are in flight):
+  [loss_time, pn_space] = GetEarliestLossTime()
+  if (loss_time != 0):
+    // Time threshold loss Detection
+    DetectLostPackets(pn_space)
+  // Don't retransmit all crypto data if a packet was just lost.
```

Comment:
This comment only makes sense for someone who followed the evolution of this algorithm. We should instead have a comment that describes when to retransmit all crypto data.